codegen/build-oas.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+version=$1 # e.g. 2024-07
+modules=("control" "data")
+
+destination="pinecone/core/openapi"
+build_dir="build"
+
+update_apis_repo() {
+	echo "Updating apis repo"
+	pushd codegen/apis
```

Comment:
Would it be useful to have something similar for making sure the `codegen/python-oas-templates` submodule is also up to date? I doubt it would have as much churn as the APIs repo, but just in case.